examples/Utils/ArgumentNames.php

```diff
@@ -65,12 +65,14 @@ final class ArgumentNames
     public const HOTEL_CENTER_ACCOUNT_ID = 'hotelCenterAccountId';
     public const KEYWORD_PLAN_ID = 'keywordPlanId';
     public const KEYWORD_TEXT = 'keywordText';
```

Comment:
The old ones are still used by other examples, right?

Reply:
Both argument names are used elsewhere in other code examples so I kept them.